### PR TITLE
Updated Loadout: US_Rangers_2010

### PR DESCRIPTION
**Pull request by AChesheireCat**
Replaced CIRAS vest with new CUP version
Replaced M4A1 w/ M4A1 Block II
Fix'd vehicle loadouts to remove FIR
Updated vehicle crew to wear UCP, now period accurate
Replaced M9 w/ M9A1
Replaced ACR helmet with new CUP OCP helmet
Replaced Uniform w/ Crye GEN 1 Multicam, now period accurate
Removed green paint from kit
Replaced GPS w/ Microdagr for increased functionality
Removed suppressors, were not issued / in serious use yet, and can be issued by mission makers if required anyways

### CODE REVIEW

File: addons/tmf_loadouts/loadouts/us_rangers_2010.hpp [hunk 1]
```diff
@@ -78,96 +76,96 @@ class baseMan {// Weaponless baseclass
...
 		"cup_acc_anpeq_15"
 	};
 	magazines[] = {
-		LIST_7("CUP_30Rnd_556x45_PMAG_QP"),
-		LIST_2("CUP_30Rnd_556x45_Emag_Tracer_Red"),
-		LIST_3("HandGrenade"),
-		"SmokeShell",
-		"ACE_HandFlare_Red"
+		LIST_8("CUP_30Rnd_556x45_Emag"),
+		LIST_2("CUP_30Rnd_556x45_PMAG_QP_Tracer_Red"),
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_rangers_2010.hpp:103]: Mixed EMAG/PMAG?

File: addons/tmf_loadouts/loadouts/us_rangers_2010.hpp [hunk 2]
```diff
@@ -176,40 +174,46 @@ class sl : ftl {
...
 
 class co : sl {
 	displayName = "Platoon Leader";
-	headgear[] = {"H_MilCap_mcamo"};
 };
 
 class fac : co {
 	displayName = "Forward Air Controller";
-	backpack[] = {"CUP_B_AssaultPack_ACU"};
+	goggles[] = {"antenna_long_two_face"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_rangers_2010.hpp:187]: Missing dependency

File: addons/tmf_loadouts/loadouts/us_rangers_2010.hpp [hunk 3]
```diff
@@ -176,40 +174,46 @@ class sl : ftl {
...
 };
 
 class ar : r {
 	displayName = "Automatic Rifleman";
-	primaryWeapon[] = {"hlc_lmg_mk46"};
-	attachment[] = {
-		"cup_acc_anpeq_15_flashlight_tan_l",
-		"cup_acc_anpeq_15"
+	vest[] = {"CUP_V_PMC_CIRAS_OD_TL"};
+	primaryWeapon[] = {"hlc_m249_pip4"};
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_rangers_2010.hpp:207]: His the M249 in the same mod as the Mk48/46?

File: addons/tmf_loadouts/loadouts/us_rangers_2010.hpp [hunk 4]
```diff
@@ -277,8 +278,7 @@ class mmgtl : aar {
 		"ItemMap",
 		"ItemCompass",
 		"ItemWatch",
-		"ACE_Vector",
-		"ItemGPS"
+		"ACE_Vector"
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_rangers_2010.hpp:281]: Adding a vector to the gunner?

**Review by Freddo3000 — CHANGES_REQUESTED** [review]
Seems like you forgot to unlist FIR from dependencies

File: addons/tmf_loadouts/loadouts/us_rangers_2010.hpp (hunk 4 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/us_rangers_2010.hpp:281]: Nope, vector existed in the last iteration. Just removing the GPS and replacing it with a microDagr in order to provide increased functionality (Vector can link with microDagr and can pre-plot distance between points on the microDagr, etc. etc.).

File: addons/tmf_loadouts/loadouts/us_rangers_2010.hpp (hunk 3 above)
Comment (AChesheireCat) [reply addons/tmf_loadouts/loadouts/us_rangers_2010.hpp:207]: Yeah, I think they're all in the M249 pack... I think. Updated anyways

File: addons/tmf_loadouts/loadouts/us_rangers_2010.hpp [hunk 5]
```diff
@@ -4,9 +4,9 @@
 //Dependencies:
 //		* CUP Units
 //		* CUP Weapons
-//		* HLC Mk46 + Mk48
+//		* HLC M249 (Mk46+Mk48) Pack
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_rangers_2010.hpp:7]: NIArms

**Review by Freddo3000 — CHANGES_REQUESTED** [review]

File: addons/tmf_loadouts/loadouts/us_rangers_2010.hpp [hunk 6]
```diff
@@ -4,7 +4,7 @@
 //Dependencies:
 //		* CUP Units
 //		* CUP Weapons
-//		* HLC M249 (Mk46+Mk48) Pack
+//		* NiArms M249 (Mk46+Mk48) Pack
```
Comment (Freddo3000) [inline addons/tmf_loadouts/loadouts/us_rangers_2010.hpp:7]: N**I**Arms

**Review by Freddo3000 — APPROVED** [review]

**Pull request merged**